blinking-Butled/pru/main0.c

```diff
@@ -0,0 +1,101 @@
+/*  Copyright (c) 2020, Code https://github.com/dinuxbg/pru-gcc-examples/blob/master/blinking-led/pru/main0.c
+ *  by  Dimitar Dimitrov is  modified by Deepankar maithani to include a button and targeting a specific pin  as output and another one as input
+ *  using  bit masking
+   All rights reserved.
+
+   Redistribution and use in source and binary forms, with or without
+   modification, are permitted provided that the following conditions are met:
+
+   * Redistributions of source code must retain the above copyright
+     notice, this list of conditions and the following disclaimer.
+   * Redistributions in binary form must reproduce the above copyright
+     notice, this list of conditions and the following disclaimer in
+     the documentation and/or other materials provided with the
+     distribution.
+   * Neither the name of the copyright holders nor the names of
+     contributors may be used to endorse or promote products derived
+     from this software without specific prior written permission.
+
+  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+  AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+  IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+  ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
+  LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+  CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+  SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+  INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+  CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+  POSSIBILITY OF SUCH DAMAGE. */
+
+
+/*
+ * IF on POCKET BEAGLE:
+ * Connect LED to 2.34
+ * Connect Button to 2.32
+ *
+ * IF on BEAGLEBONE BLACK OR WIRELESS
+ * connect LED to P9.27
+ * Connect Button to 9.30
+ *
+ *
+ * The Button connection Diagram   3.3v-------___ ------------------|----------------------------^^^^------GND
```

Comment:
Please use fritzing to draw a schematic, and add it to README.md . Or if you insist on ASCII, then please try to make it smaller and more readable.